pom.xml

```diff
@@ -356,24 +356,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>3.1.1</version>
-        <configuration>
-          <configLocation>checkstyle.xml</configLocation>
-          <linkXRef>false</linkXRef>
-          <violationSeverity>warning</violationSeverity>
-          <includeTestSourceDirectory>true</includeTestSourceDirectory>
-        </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>check</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
This is actually a useful plugin and is used in [quite a few other repositories](https://github.com/search?p=3&q=org%3Afolio-org+maven-checkstyle-plugin&type=Code) on the FOLIO project. So please add it back and make the changes that it recommends.

Most of the complaints are things like "indent level was 6 - it should be 8". That might seem odd but it's noticing that something like this:
```
    final Checkout checkout = Checkout.builder()
      .scRenewalPolicy(FALSE) // Indented 2 spaces (which is a "level")
```
which should be instead:
```
    final Checkout checkout = Checkout.builder()
         .scRenewalPolicy(FALSE) // Indented 4 spaces (two levels)
```
I think the other things that flags are line length being <= 100, so also easy to fix.

Reply:
The `checkstyle.xml` file in the project root is worth a look. The rule for continuing lines with 4 spaces rather than 2 is here: https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent.

Reply:
Added , also code indentation fixed accordingly please check